Realm/Realm/Sync/ProgressNotifications/ProgressMode.cs

```diff
@@ -25,15 +25,12 @@ public enum ProgressMode
     {
         /// <summary>
         /// The callback will be called forever, or until it is unregistered by disposing the subscription token.
-        /// Notifications will always report the latest number of transferred bytes, and the most up-to-date number of
-        /// total transferable bytes.
         /// </summary>
         ReportIndefinitely,
 
         /// <summary>
-        /// The callback will, upon registration, store the total number of bytes to be transferred. When invoked, it will
-        /// always report the most up-to-date number of transferable bytes out of that original number of transferable bytes.
-        /// When the number of transferred bytes reaches or exceeds the number of transferable bytes, the callback will
+        /// The callback will be active until the current transferable bytes are transferred. When invoked, it will always report the
```

Comment:
it's a bit weird now to refer to transferred/transferable after these fields were removed. Also, it makes no sense any more for download direction. Probably, a good idea is to rephrase it in a way that it refers to snapshot committed version for upload on callback registration (that is essentially transferable bytes), and for download it is whatever server decides for the outgoing or next batch of changesets to catch up the client to the latest version.

Reply:
@kiburtse I agree with you, it's kinda imprecise. 
Would you think the following will make sense?

```
The callback will be active:
- For uploads, until all the unsynced changesets available at the moment of the registration of the callback are sent to the server.
- For downloads, until the client catches up to the current data (available at the moment of registration) or the next batch of changesets sent from the server.
```

Reply:
@nirinchev I'll fix the other two small things. I suppose this is the latest thing to agree on. Do you have input on this?